src/main/java/mat/server/CQLConstantServiceImpl.java

```diff
@@ -175,6 +161,35 @@ public CQLConstantContainer getAllCQLConstants() {
         return cqlConstantContainer;
     }
 
+    private boolean isFhirEditEnabled() {
+        return featureFlagService.findFeatureFlags().getOrDefault(FeatureFlagConstant.FHIR_EDIT, false);
+    }
```

Comment:
Would it make sense to move these kind of checks into the FeatureFlagService? Perhaps, `featureFlagService.isFhirEditEnabled()`.

Reply:
Well, maybe you're right.
Need to look into that.